pages/shared/groupWorkInitial.ejs

```diff
@@ -0,0 +1,33 @@
+<div class="container-fluid">
+  <div class="row">
+      <% if (permissions.student_authz_create) { %>
+      <div class="col-sm bg-light py-4 px-4 border">
+        <form id="create-form" name="create-form" method="POST">
+          <input type="text" class="form-control" name="groupName" placeholder="Group Name">
```

Comment:
Placeholder text should not be used as a replacement for a label: https://www.w3.org/WAI/tutorials/forms/instructions/#placeholder-text

Here it would be best to have an actual form label like "Group name" (note the lowercase "n" in "name").